react/helpers/ProductKitHelper.js

```diff
@@ -0,0 +1,80 @@
+import { findIndex, propEq } from 'ramda'
+
+/**
+ * Extract the Kit Items of a Product Kit
```

Comment:
As this component is already in the helpers folder, I think you could name it 'index.js' and import it with

`import { whatever } from './helpers'`